code/modules/mob/living/simple_animal/hostile/droid.dm

```diff
@@ -0,0 +1,35 @@
+/mob/living/simple_animal/hostile/droid
+	name = "Syndicate Droid Y-283"
+	desc = "A syndicate produced droid, fires weak laser beams and looks quite fragile. A robust toolbox would be useful right now."
+	icon = 'icons/mob/hivebot.dmi'
+	icon_state = "SecBot"
+	icon_living = "SecBot"
+	icon_dead = "SecBot"
+	maxHealth = 30
+	health = 30
+	speed = 3
+	ranged = 1
+	retreat_distance = 1
+	minimum_distance = 3
+	projectiletype = /obj/item/projectile/temp/droid
+	projectilesound = 'sound/weapons/laser.ogg'
+	gold_core_spawnable = 1
+	aggro_vision_range = 9
+	idle_vision_range = 2
+	turns_per_move = 5
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 1, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
```

Comment:
max tox should probably be zero, don't know why a robot that doesn't breathe would get hurt by breathing in plasma